src/itest/java/uk/gov/companieshouse/charges/data/controller/ChargesControllerTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.companieshouse.charges.data.controller;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import uk.gov.companieshouse.charges.data.service.ChargesService;
+
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
+
+public class ChargesControllerTest {
+
+    @Mock
+    ChargesService mockChargesService;
+
+    @InjectMocks
+    ChargesController chargesController;
+
+    private MockMvc mockMvc;
+
+    @BeforeEach
+    void setUp() {
```

Comment:
Not sure adding Charges controller was in scope for this story, we can probably remove it for now and add each end point as per story by referring to private-api-sdk model classes

Reply:
ok, will remove the change